product-updates/syskit-security-manager-1-release-note.md

```diff
@@ -1,6 +1,59 @@
 ---
-title: SysKit Security Manager - Release Note
+title: SysKit Security Manager - SharePoint and Office 365 security reporting and management tool
 author: Petra Filipi     
 date: 14/03/2018 
 description: This article describes what is this new SharePoint and Office 365 security reporting and management tool bringing in the first release.
----
\ No newline at end of file
+---
+SysKit Security Manager is centralized __SharePoint and Office 365 security reporting and management tool__. It was created to help SharePoint site collection administrators with controlling the security in their SharePoint and Office 365 environment. Let's see what this version of the tool is bringing to the table.
+
+[Try SysKit Security Manager Free](https://www.syskit.com/products/security-manager/download/)
+
+__Product version:__ 1.0.0  
+__Build number:__  407   
+__Release date:__ Apr 11, 2018  
+
+## Site Collections Screen – Explore and manage SharePoint permissions (Online & On-Premise)
+Good news! Managing SharePoint permissions just got easier with SysKit Security Manager. The tool offers the following management actions:
+* __Grant Permissions__
+* __Break/Restore Inheritance__
+* __Create SharePoint Group__
+* __Edit Group Settings__
+* __Add to Group/Copy to Group__
+* __Add Members__
+* __Move to Group__
+* __Edit Permissions__
+* __Remove Permissions__
+* __Clone Permissions__
+* __Transfer Permissions__
+* __Delete User__
+* __Delete Group__
+
+Read more about [Site Collections](#internal/get-to-know-security-manager/site-collections-screen) screen possibilities and management options.
+
+## Items with Unique Permissions 
+Detection of unique permissions at the item level and deep nested SharePoint structure can be challenging task.
+This report helps you find out folders and files which have unique permissions. Drill to Report action enables you to list all permissions of the items with broken inheritance. 
+
+## External Users & Anonymous Access Links 
+If your site collection is enabled for sharing with external users or sharing by using anonymous access links is allowed, these two reports can help you increase data security and compliance.
+* __External Users on Site Collection__ – Shows all of the external users that have accessed your site collection.
+* __Anonymous Access Links__ – Lists all the items that are shared to anonymous users with anonymous access links.
+
+## Access Requests
+Review the list of people requesting access and manage access requests.
+
+## Office 365 Groups Management
+Get an overview of your connected Office 365 Groups add manage group settings, permissions and membership from a single interface.
```

Comment:
...Groups, add or manage groups and group settings, ....
uglavnom ovaj 'add manage' je nekompletan